triggers.md

```diff
@@ -177,6 +178,29 @@ Note that if a page is truly unsure whether a given URL will be prerendered into
 
 However, in implementations such as Chromium that need the target hint, this will prerender the page twice, and thus use twice as many resources. So this is best avoided if possible.
 
+### Explicit referrer policy
+
+By default, the referring document's referrer policy is used for the speculative request. If the future navigation will use a different referrer policy (e.g. using `rel="noreferrer"`), the policy to use for the speculative request can be specified in a rule using the `"referrer_policy"` key and the desired referrer policy string.
```

Comment:
The "If the future navigation will use a different referrer policy" bit seems to contradict the note at the bottom. I think it scans better if you simply say "The policy to use for the speculative request can be specified...".

Reply:
Done